package.json

```diff
@@ -57,6 +57,7 @@
   "scripts": {
     "ui": "cross-env PORT=6002 react-scripts start",
     "server": "PORT=6001 node server",
+    "server-windows": "SET PORT=6001 && node server.js",
```

Comment:
Okay solution for now but I think even better would be a `cross-env` implementation (https://www.npmjs.com/package/cross-env)

Reply:
I did not know about that implementation, it works perfectly! Thanks.